packages/orbit-components/src/Wizard/Wizard.ct.tsx

```diff
@@ -1,16 +1,28 @@
 import * as React from "react";
 import { test, expect } from "@playwright/experimental-ct-react";
+import { defaultTokens } from "@kiwicom/orbit-design-tokens";
 
 import RenderInRtl from "../utils/rtl/RenderInRtl";
 import { WizardStory, WizardStoryHover } from "./Wizard.ct-story";
 
 test.describe("visual Wizard", () => {
   async function maybeHoverByText(component, text: string) {
+    const { breakpointMediumMobile } = defaultTokens;
+
     if ((await component.getByText(text).count()) === 0) {
       return Promise.resolve();
     }
 
-    return component.getByText(text).hover();
+    const element = component.getByText(text);
+    await element.hover();
+
+    // Skip hover assertion for Small Mobile & Medium Mobile viewports
+    const viewport = component.page().viewportSize();
+    const isMobile = viewport && viewport.width <= breakpointMediumMobile; // typical mobile breakpoint
+    if (!isMobile) {
+      await expect(element).toHaveClass(/hover/);
```

Comment:
I added this part of the code to avoid test flakiness in this PR. 🤔  I checked and tested the component locally, and it's working well.

Flakiness issue:
```
row hover completed step
```

![image](https://github.com/user-attachments/assets/f912c0cc-2de1-44bc-89d1-bc79f09ba829)


![image](https://github.com/user-attachments/assets/08f5b671-a80c-4342-8a69-7b5e3d4f07f2)